bot.rb

```diff
@@ -41,9 +41,26 @@ def myself
       client.user.login
     end
 
+    def comment_and_close(body, issue)
+      client.add_comment(SLUG, issue.number, body.join("\n\n"))
+      client.close_issue(SLUG, issue.number)
+      client.add_labels_to_an_issue(SLUG, issue.number, AUTO_CLOSED)
+    end
+
+    def process_inactive_migrated(issue)
+      puts "https://github.com/#{SLUG}/issues/#{issue.number} (#{issue.title}) has been inactive since monorepo migration, closing now"
+      body = []
+      body << "There hasn't been any activity on this issue recently. Due to the high number of incoming issues, we are cleaning up some of the older issues as many of them have been resolved with recent updates."
+      body << "Please make sure to update to the latest `fastlane` version and check if that solves the issue."
+      body << "We are going to close this issue, but please feel free to [open a new one](https://github.com/fastlane/fastlane/issues/new) if you are still experiencing this problem 👍"
+      comment_and_close(body, issue)
+    end
     # Responsible for commenting to inactive issues
     def process_inactive(issue)
-      return if issue.comments == 0 # we haven't replied yet :(
+      is_inactive_migrated_issue = issue.user.login == 'fastlane-bot' && !!client.issue_comments(SLUG, issue.number).find { |c| c.user.login != 'fastlane-bot' }
```

Comment:
Not all issues were migrated by the same bot, as we had to create multiple bots to migrate all the issues. So this will not work for all issues